sql/aggregate-group-by-functions.md

```diff
@@ -90,6 +113,9 @@ from tbl_name
 group by id, val;
 ```
 
-## Detection of functional dependence
+## Unsupported aggregate functions
+
+The following aggregate functions are currently unsupported in TiDB.  You can track our progress in [TIDB #7623](https://github.com/pingcap/tidb/issues/7623):
```

Comment:
- Remove one extra space before "You can". Only leave one.
- TIDB 7623 -> TiDB 7623